cmd/edenRol.go

```diff
@@ -0,0 +1,141 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Insei/rolgo"
+	"github.com/lf-edge/eden/pkg/defaults"
+	"github.com/lf-edge/eden/pkg/utils"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"path"
+)
+
+var (
+	rolProjectID    string
+	rolRentName     string
+	rolModel        string
+	rolManufacturer string
+	rolRentID		string
+	rolIPXEUrl		string
+)
+
+var rolCmd = &cobra.Command{
+	Use:   "rol",
+	Short: `Manage devices in Rack Of Labs`,
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		if err := rootCmd.PersistentPreRunE(cmd, args); err != nil {
+			return err
+		}
+		assignCobraToViper(cmd)
+
+		_, err := utils.LoadConfigFile(configFile)
+		if err != nil {
+			return err
+		}
+		return nil
+	},
+}
+
+var rolRentCmd = &cobra.Command{
+	Use:   "rent",
+	Short: "Manage device rents",
+	Long:  `Manage device rents`,
+}
+
+var createRentCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a new device rent",
+	Long:  `Create a new device rent`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		assignCobraToViper(cmd)
+
+		client, err := rolgo.NewClient()
```

Comment:
Do we need some token or password to access the backend?

Reply:
Yep, in ENV, we need to set ROL_API_URL and ROL_API_KEY.